internal/app/routes/userRoutes/userRoutes.go

```diff
@@ -0,0 +1,12 @@
+package userRoutes
+
+import (
+	"github.com/go-chi/chi"
+	"github.com/gzcharleszhang/course-planner/internal/app/handlers/userHandlers"
+)
+
+func InitUserRoutes(r chi.Router) {
+	r.Route("/users", func(r chi.Router) {
+		r.Post("/", userHandlers.NewUserHandler)
```

Comment:
@infinitecold it might look kinda bare right now, but once we starting to add more routes, it won't look nice with addtional logic.

Reply:
makes sense